html/images/os/adva.svg

```diff
@@ -0,0 +1 @@
fill:#3950a3;stroke:#305e84;stroke-miterlimit:10;}.cls-4{fill:#004588;}.cls-5{fill:#28e87e;}</style></defs><title>adva</title><line class="cls-1" x1="287.28" y1="281.75" x2="322.32" y2="281.75"/><line class="cls-1" x1="289.03" y1="288.25" x2="326.32" y2="288.25"/><path class="cls-1" d="M328.32,309.5" transform="translate(-2)"/><line class="cls-1" x1="328.07" y1="309.5" x2="301.32" y2="282.75"/><path class="cls-2" d="M75.53,40.57A106.34,106.34,0,0,0,52.79,98.65h64.63a43.93,43.93,0,0,1,3.82-10.25Z" transform="translate(-2)"/><path class="cls-2" d="M159.28,0a106.29,106.29,0,0,0-66.5,23.26l45.33,47.43a44,44,0,0,1,65.14,28h62.52A106.82,106.82,0,0,0,159.28,0Z" transform="translate(-2)"/><path class="cls-2" d="M122,130.3a48.34,48.34,0,0,1-3.07-7H53.82a116.68,116.68,0,0,0,23.42,54.63Z" transform="translate(-2)"/><path class="cls-2" d="M201.76,123.28c-6.17,17.54-22.36,30.09-41.42,30.09a42.54,42.54,0,0,1-20.92-5.51L94.94,194a103.87,103.87,0,0,0,64.34,22.37c53.2,0,97.29-40.31,105.46-93.07Z" transform="translate(-2)"/><path class="cls-3" d="M212,327.17" transform="translate(-2)"/><path class="cls-3" d="M251,249.26" transform="translate(-2)"/><path class="cls-4" d="M38.15,250.94H63.46L81,298.56H12.84Zm-.7,39.72H57.59l-8.53-24.27Z" transform="translate(-2)"/><polygon class="cls-4" points="30.14 304.37 7.83 304.37 0 318.7 23.14 318.7 30.14 304.37"/><polygon class="cls-4" points="80.77 304.37 58.47 304.37 62.1 318.7 85.94 318.7 80.77 304.37"/><path class="cls-3" d="M192.24,324.81" transform="translate(-2)"/><path class="cls-3" d="M165.85,248" transform="translate(-2)"/><path class="cls-4" d="M269.77,250.94h25.31l17.49,47.62H244.46Zm-.7,39.72h20.14l-8.53-24.27Z" transform="translate(-2)"/><polygon class="cls-4" points="261.76 304.37 239.45 304.37 231.62 318.7 254.76 318.7 261.76 304.37"/><polygon class="cls-4" points="312.39 304.37 290.09 304.37 293.72 318.7 317.56 318.7 312.39 304.37"/><polygon class="cls-4" points="183.89 303.81 220.74 303.81 213.26 318.86 189.08 318.86 183.89 303.81"/><polygon class="cls-4" points="225.53 250.94 247.19 250.94 222.96 298.86 204.05 298.86 225.53 250.94"/><polygon class="cls-4" points="182.21 298.86 202.19 298.86 187.56 250.94 166.21 250.94 182.21 298.86"/><path class="cls-4" d="M165.85,277.16a25.94,25.94,0,0,0-5-15.39c-8.21-11.32-22.9-10.84-28.15-10.84H97.25v47.62H118V265.83H132.7c10,0,11.72,12.42,11.64,16.26-.21,11.33-4.09,16.47-4.09,16.47h23.18S166.13,286.77,165.85,277.16Z" transform="translate(-2)"/><path class="cls-1" d="M163.44,298.56" transform="translate(-2)"/><path class="cls-1" d="M132.7,250.94c0,13.29,14.29,24,31.94,24" transform="translate(-2)"/><path class="cls-4" d="M134.1,318.86H97.25V304.37h62.9S149.91,318.86,134.1,318.86Z" transform="translate(-2)"/><path class="cls-1" d="M134.1,304.37" transform="translate(-2)"/><path class="cls-1" d="M160.15,304.37" transform="translate(-2)"/><path class="cls-1" d="M134.1,304.37" transform="translate(-2)"/><path class="cls-5" d="M169.77,314.16" transform="translate(-2)"/></svg>
```

Comment:
OS logos should be Y x Y (square), this is slightly different in width. Can you stretch it out?